static/app/components/replays/videoReplayer.tsx

```diff
@@ -397,29 +398,35 @@ export class VideoReplayer {
 
   /**
    * Shows the video -- it is assumed that it is preloaded. Also
-   * hides the previous video, there should not be a reason we show
-   * a video and not hide the previous video, otherwise there will
+   * hides all other videos, otherwise there will
    * be multiple video elements stacked on top of each other.
    */
   protected showVideo(nextVideo: HTMLVideoElement | undefined): void {
     if (!nextVideo) {
       return;
     }
 
-    // This is the soon-to-be previous video that needs to be hidden
-    if (this._currentVideo) {
-      this._currentVideo.style.display = 'none';
-      // resets the soon-to-be previous video to the beginning if it's ended so it starts from the beginning on restart
-      if (this._currentVideo.ended) {
-        this.setVideoTime(this._currentVideo, 0);
+    for (const video of this._videos) {
```

Comment:
how about destructuring to get named vars:
```suggestion
    for (const [index, videoElem] of this._videos) {
```
and then `video[0]` would be `index` and `video[1]` would be `videoElem` now